languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/blacklist.txt

```diff
@@ -0,0 +1,64 @@
+# This file records dictionary entries frequently added by users that we DO NOT
+# want to add to our own spelling dictionary.
+#
+# This is here to make it easier to identify *new* candidates for the spelling
+# dictionary without having to comb through the same list of the usual suspects
+# week in, week out.
+#
+# For now this is used by @p-goulart's script, hopefully we'll also have a GHA
+# workflow to check.
+#
+# Our list is structured thus:
+# entry	comments
+# other entry	more comments
+# separated by a tab char.
+
+LTDA	requires a full stop
+Academy	foreign term
+academy	foreign term
+action	foreign term
+All	foreign term
+all	foreign term
+AND	foreign term
+and	foreign term
+American	foreign term
+Account	foreign term
+account	foreign term
+at	foreign term
+Back	foreign term
+back	foreign term
+Bank	foreign term
+Christi	'Corpus Christi' is already a valid entry
+Brazil	foreign term
+Iphone	wrong casing
+email	missing hyphen
+ebook	missing hyphen
+netflix	wrong casing
+tiktok	wrong casing
+Tiktok	wrong casing
+python	wrong casing
+black	foreign term
+book	foreign term
+boot	foreign term
+by	foreign term
+teixeira	wrong casing
+food	foreign term
+Food	foreign term
+Forms	foreign term
+forms	foreign term
+Front	foreign term
+front	foreign term
+Full	foreign term
+full	foreign term
+from	foreign term
+Gonzalez	wrong spelling in foreign language
```

Comment:
I wonder if some of the foreign terms shouldn't be added to the dictionary anyway. For example, I imagine `book` might be used a lot (or relatively often) in the marketing/influencer/photography industries. It's pretty common to come across sentences like `Gostaria de fazer um book de fotos?`

Or proper names like `Gonzalez` could have multiple spellings, so shouldn't we have as many of them as possible be in the dictionary?